### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by mell122**

### CODE REVIEW

**Comment by mell122** [comment]
update go.yml

File: .github/api-gateway/CHANGELOG.md
Comment (miladhzzzz) [inline .github/api-gateway/CHANGELOG.md:1]: i think you accidentally copied the whole api-gateway directory to .github directory!

File: .github/workflows/go.yml [hunk 1]
```diff
@@ -25,9 +25,47 @@ jobs:
       working-directory: ./api-gateway
       run: |
           ls -la
+    
+    - name: Build
```
Comment (miladhzzzz) [inline .github/workflows/go.yml]: this workflow runs only ls -la on the current directory that you're in not building any go binaries. pay attention to the run part!!

File: .github/workflows/go.yml [hunk 2]
```diff
@@ -25,9 +25,47 @@ jobs:
...
+      working-directory: ./monitoring-service
+      run: |
+          ls -la
+
+    - name: Build
+      working-directory: ./notify-service
+      run: |
+          ls -la
+
```
Comment (miladhzzzz) [inline .github/workflows/go.yml]: you are just running ls -la over and over again ls list's the content of the directory! and you need to make a seprate yaml file for every service and not make a multi step single file.

File: .github/api-gateway/utils/token.go
Comment (miladhzzzz) [inline .github/api-gateway/utils/token.go:1]: remove the api-gateway directory from .github directory.

**Review by miladhzzzz — COMMENTED** [review]
thank you for the update i pointed out the problems i found with this pull request in my review and you need to provide more detail in your pr to describe exactly what your doing.
please refer to this pr to get a nice template on how to write descriptions: 
https://github.com/kubernetes/kubernetes/pull/122350

**Pull request closed without merging**